R/plots.R

```diff
@@ -1,12 +1,18 @@
 create_main_projection_chart <- function(dat, horizon) {
-  conv_lab <- \(x) {
+  convert_year_labels <- function(x) {
     paste0(format(x, "%Y"), "/", (as.integer(format(x, "%Y")) + 1) %% 100)
   }
-  fy_as_date <- \(x) as.Date(sub("_[0-9]{2}$", "-01-01", x))
+  fy_as_date <- \(x) as.Date(sub("_[0-9]{2}$", "-04-01", x))
```

Comment:
Setting the date values for the x-axis to 1 April rather than 1 January makes no difference whatsoever to the plots, but it just felt like the right thing to do as we are working with financial years.